__tests__/data/shopify_source_output.json

```diff
@@ -501,7 +502,7 @@
       "SHOPIFY": true
     },
     "type": "track",
-    "event": "fulfillments_update",
+    "event": "Fulfillments Update",
```

Comment:
This will be a breaking change, right?

Reply:
Only in terms of event names will address that in change-log